typescript.js

```diff
@@ -0,0 +1,408 @@
+/* eslint sort-keys: ["error", "asc"], quote-props: ["error", "consistent"] */
+/* eslint-disable sort-keys */
+
+module.exports = {
+    plugins: ["@typescript-eslint"],
```

Comment:
please check what you need to install to use the plugin and the parser and install and `--save` it. Otherwise it won't work.